enterprise/static/enterprise/js/enterprise_login_page.js

```diff
@@ -0,0 +1,46 @@
+function redirectToURL(redirectURL) {
+    location.href = redirectURL;
+}
+
+function setupFormSubmit() {
+    $('#enterprise-login-form').submit(function(event){
+        event.preventDefault();
+        var enterpriseSlug = $("#id_enterprise_slug").val();
+
+        $("#activate-progress-icon").removeClass("is-hidden");
+        $("#enterprise-login-submit").attr("disabled", true);
+
+        $.ajax({
+            url : "/enterprise/login",
+            method : "POST",
+
+            beforeSend: function (xhr) {
+                xhr.setRequestHeader("X-CSRFToken", $.cookie("csrftoken"));
+            },
+
+            data : {
+                enterprise_slug : enterpriseSlug
+            },
+
+            success: function(data) {
+                redirectToURL(data.url);
```

Comment:
Should we display some error to user we will get this URL empty. for example in the case if import error

Reply:
I have handled that case in the backend part. Thanks for this.